assets/css/styleguide.scss

```diff
@@ -377,4 +377,12 @@ header[role="banner"] {
 
 .download-code {
   padding-top: 3em;
+}
+
+// Search bar grid --------- //
+
+.preview-search-bar {
+  .usa-grid-box {
+    padding: 0;
+  }
 }
```

Comment:
@maya @msecret we've been adding new lines to the end of every file and think it'd be good to stay consistent in this practice.